src/templates/product.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="{{ language }}">
+<html lang="{{ options.language }}">
```

Comment:
do `language` and `toc` need to be nested inside `options.`? Makes everything a bit long to read.